testing/unit-testing-jest/docs/introduction.md

```diff
@@ -1,10 +1,50 @@
-## Whats is testing and why is important?
+# Introduction
+### General concept and benefits
+
+Essentially, a unit test is a method that instantiates a small portion of our application and checks its behavior independently of other parts.
+
 Testing is a method to check if the code is working as expected. Testing the code helps to detect bugs on time and prevent the appearance of new bugs in the future. Also, testing gives confidence to our code. We can expect that they don't find problems in QA tests.
 
-## Benefits
 * Fewer bugs in production
 * Fewer code regressions
 * More maintainable code
 * Faster refactoring and upgrades
 * Less dead code
 * You can see more on this [article](https://jestjs.io/docs/using-matchers).
+
+
+### Principles 
+
+**Easy to write:**  It should be easy to code all of those test routines without enormous effort.
```

Comment:
It is minimal, but after a colon, it is always lowercase, only in some cases it is capitalized.